app/assets/stylesheets/application.css

```diff
@@ -20,4 +20,6 @@
  *= require product_slickgrid
  *= require kubernetes-topology-graph/topology-graph
  *= require jquery-ui-miq-theme/jquery-ui-1.9.2.custom
+ *= require angular-patternfly
+ *= require dashboard
```

Comment:
container_providers_dashboard would be a better name given that there are already several dashboards in the system, and there will be more.